scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts

```diff
@@ -229,6 +231,13 @@ export function ContentProfileSchemaEditor(content, metadata, vocabularies) {
                  * @param {String} id the key of the field to toggle.
                  */
                 scope.toggle = (schema, order, position) => {
+                    let newOrder = order;
+
+                    if (scope.model.editor[schema.key].enabled) {
+                        notify.error(gettext('This item is already added in {{section}} fields.',
```

Comment:
How can the error happen? Shouldn't the dropdown only contain the values user is allowed to select?

Reply:
yep, the dropdown should contain the allowed values only.

Reply:
So this should never happen because it's already validated before displaying the dropdown. In such cases, it's better to simply throw an error to indicate to other developers that this not expected behavior and not bother the translators with translating texts which will probably never be used.